admin/sql/create_tables.sql

```diff
@@ -18,4 +18,22 @@ CREATE TABLE listen (
   raw_data        JSONB
 );
 
+CREATE TABLE token (
+     id               SERIAL,
```

Comment:
If these tables are only used for our compatibility API, I'd like to separate them. Either give them a name prefix (`compat`?) or put them in a separate schema.